packages/runtime/container-runtime/src/opLifecycle/opCompressor.ts

```diff
@@ -23,6 +26,20 @@ export class OpCompressor {
     }
 
     public compressBatch(batch: IBatch): IBatch {
+        if (batch.contentSizeInBytes > maxSupportedSizeInBytes) {
```

Comment:
can we avoid this hard-coded length limit? I.e. go through normal path, and if it fails (maybe check for specific error string), and the size is large, use more telling error message instead? That way, if JS heap max size improves over time, we are not left with artificial limit.

That all said, sizes of that magnitude are egregious.

Reply:
good point